code-samples/serving/hello-world/helloworld-python/README.md

```diff
@@ -35,7 +35,7 @@ cd knative-docs/code-samples/serving/hello-world/helloworld-python
    cd app
    ```
 
-1. Create a file named `app.py` and copy the following code block into it:
+2. Create a file named `app.py` and copy the following code block into it:
```

Comment:
AFAIK the numbers shouldn't need to be specifically called out like this, we should be able to use `1.` for each of them, which is easier for future maintenance: https://www.markdownguide.org/basic-syntax/#lists-1

At https://github.com/knative/docs/tree/main/code-samples/serving/hello-world/helloworld-python it looks like it only caused rendering issues because of the broken `=== "kn"` bits, etc. Please leave this as `1.`